uiautomationutilities/src/main/java/com/microsoft/identity/client/ui/automation/broker/AbstractTestBroker.java

```diff
@@ -53,6 +55,25 @@
  */
 public abstract class AbstractTestBroker extends App implements ITestBroker {
 
+    private final static String TAG = AbstractTestBroker.class.getSimpleName();
+
+    @Override
+    public void uninstall() {
+        super.uninstall();
+
+        if (isInstalled()) {
+            // The broker app will still be installed on the device if it is enabled
+            // as a device admin. In this case, we need to disable the admin and then
+            // uninstall.
+            Log.w(TAG, "Unable to uninstall broker " + getAppName() + " from device..." +
+                    "the broker is potentially enabled as an active device admin.");
+            Log.i(TAG, "Disabling admin for " + getAppName());
+            TestContext.getTestContext().getTestDevice().getSettings().disableAdmin(getAdminName());
+            Log.i(TAG, "Reattempting uninstall for " + getAppName());
+            super.uninstall();
+        }
```

Comment:
This is just refactor....just relocating this code here from another class where it was previously located.